apps/admin-portal/src/components/claims/dynamic-fields.tsx

```diff
@@ -39,6 +39,8 @@ interface DynamicFieldPropsInterface {
     valueName: string;
     keyRequiredMessage: string;
     valueRequiredErrorMessage: string;
+    requiredField?: boolean | undefined;
```

Comment:
`undefined` is redundant here. If `requiredField` is not passed to the component, it is `undefined` by default.